skimage/transform/tests/test_radon_transform.py

```diff
@@ -80,21 +81,31 @@ def check_radon_center(shape, circle):
 
 shapes_for_test_radon_center = [(16, 16), (17, 17)]
 circles_for_test_radon_center = [False, True]
+dtype_for_test_radon_center = [np.float64, np.float32, np.uint8, bool]
+preserve_range_for_test_radon_center = [False, True]
 
 
-@testing.parametrize("shape, circle",
+@testing.parametrize("shape, circle, dtype, preserve_range",
```

Comment:
this is extremely confusing. You can just have multiple `parametrize` decorators which becomes much more readable. One decorator per parameter. itertools product is pretty obscur for science programming.

Reply:
I agree, just wanted to keep the same test structure as before. Can I also rename the parameter lists :smile: ?

Reply:
you can just remove the fact that they are defined in the first place.
The list is easily defined inline